__mocks__/react-map-gl.tsx

```
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export const Map = ({ children }: { children: React.ReactNode }) => (
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)